src/components/FilterGroup.tsx

```diff
@@ -69,6 +72,9 @@ export function FilterGroup({
     };
   }, [customCssClasses]);
 
+  const limited = filterOptions.length > showMoreLimit;
```

Comment:
Super subjective, but since this is a boolean it might be a tad more legible for the future if it were called `isLimited`-- but its tiny haha